crates/bitwarden/src/mobile/mod.rs

```diff
@@ -1,6 +1,6 @@
-#[cfg(feature = "internal")]
 pub mod crypto;
 pub mod kdf;
+pub mod tool;
```

Comment:
Any reason to call the modules `tool` instead of `tools`?

Reply:
The non module moduled is called `tool`. But other than that no.